docs/snippets/examples/reliable-keyspace-notification.py

```diff
@@ -0,0 +1,3 @@
+GearsBuilder() \
+.foreach(lambda x: execute('XADD', "notifications-stream", '*', *sum([[k,v] for k,v in x.items()],[]))) \
+.register(prefix="person:*", eventTypes=['hset', 'hmset'])
```

Comment:
What about the other hset commands? del? rename?

Reply:
it's just an example, similar to using `prefix="person:*"`